tests/benchmarking/test_benchmarking.py

```diff
@@ -67,175 +67,68 @@ def test_build_django_from_cache(benchmark):
         benchmark(fn)
 
 
+def test_find_descendants(large_graph, benchmark):
+    result = benchmark(large_graph.find_descendants, "mypackage")
+    assert len(result) == 28222
+
+
+def test_find_downstream_modules(large_graph, benchmark):
+    result = benchmark(large_graph.find_downstream_modules, DEEP_LAYERS[0], as_package=True)
+    assert len(result) == 80
+
+
+def test_find_upstream_modules(large_graph, benchmark):
+    result = benchmark(large_graph.find_upstream_modules, DEEP_LAYERS[0], as_package=True)
+    assert len(result) == 2159
+
+
+def test_find_shortest_chain(large_graph, benchmark):
+    result = benchmark(large_graph.find_shortest_chain, DEEP_LAYERS[0], DEEP_LAYERS[1])
+    assert result is not None and len(result) == 5
+
+
+def test_find_shortest_chain__no_chain(large_graph, benchmark):
+    result = benchmark(
+        large_graph.find_shortest_chain,
+        DEEP_LAYERS[0],
+        "mypackage.data.vendors.4053192739.6373932949",
+    )
+    assert result is None
+
+
+def test_find_shortest_chains(large_graph, benchmark):
```

Comment:
Mind grouping these tests for the same method into a `TestFindShortestChains` class?